src/engine/renderer/tr_decals.cpp

```diff
@@ -131,8 +131,11 @@ static bool MakeTextureMatrix( vec4_t texMat[ 2 ], vec4_t projection, decalVert_
 		}
 	}
 
-	texMat[ 0 ][ 3 ] = a->st[ 0 ] - DotProduct( pa, texMat[ 0 ] );
-	texMat[ 1 ][ 3 ] = a->st[ 1 ] - DotProduct( pa, texMat[ 1 ] );
+	for ( int k : {0, 1} )
```

Comment:
Wonder what the problem is and why this doesn't seem to have anything to do with the commit message?